bugbug/models/regressionrange.py

```diff
@@ -69,6 +69,9 @@ def get_labels(self):
         classes = {}
 
         for bug_data in bugzilla.get_bugs():
+            if not bug_data["regressions"]:
```

Comment:
As you can see in the [bugzilla docs](https://bmo.readthedocs.io/en/latest/api/core/v1/bug.html), the `regressions` field lists the IDs of bugs that are introduced by this bug. Thus, this bug is a regressor, not a regression.

Regressions are the bugs that have the `regression` keyword.

Reply:
Thank you for the review. I made some updates 🙂